### PR TITLE
Custom commands

### PR DESCRIPTION
**Pull request by JordyGit**

### CODE REVIEW

File: release-notes/Cloud/DCP_change_log.md [hunk 1]
```diff
@@ -48,6 +56,25 @@ To get started, you can find several example Automation scripts with more inform
 
 After you have made sure that the [prerequisites](https://github.com/SkylineCommunications/ChatOps-Extensions/blob/main/ChatIntegrationExamples/README.md#usage) are in place, you can deploy [the Chat Integration examples](https://github.com/SkylineCommunications/ChatOps-Extensions/blob/main/ChatIntegrationExamples/README.md#getting-started) to your DataMiner System and immediately try out these examples.
 
+#### 6 February 2024 - New feature - Custom commands executed with the DataMiner bot now support adding buttons using Adaptive Cards to the output [ID_38701]
```
Comment (JordyGit) [inline release-notes/Cloud/DCP_change_log.md:59]:
```suggestion
#### 6 February 2024 - New feature - Custom commands executed with the DataMiner bot now support adding buttons to the output using Adaptive Cards [ID_38701]
```

File: release-notes/Cloud/DCP_change_log.md [hunk 2]
```diff
@@ -48,6 +56,25 @@ To get started, you can find several example Automation scripts with more inform
 
 After you have made sure that the [prerequisites](https://github.com/SkylineCommunications/ChatOps-Extensions/blob/main/ChatIntegrationExamples/README.md#usage) are in place, you can deploy [the Chat Integration examples](https://github.com/SkylineCommunications/ChatOps-Extensions/blob/main/ChatIntegrationExamples/README.md#getting-started) to your DataMiner System and immediately try out these examples.
 
+#### 6 February 2024 - New feature - Custom commands executed with the DataMiner bot now support adding buttons to the output using Adaptive Cards [ID_38701]
```
Comment (JordyGit) [inline release-notes/Cloud/DCP_change_log.md:59]:
```suggestion
#### 6 February 2024 - New feature - Custom commands executed with the DataMiner bot now support adding buttons to the response using Adaptive Cards [ID_38701]
```

File: release-notes/Cloud/DCP_change_log.md [hunk 3]
```diff
@@ -48,6 +56,25 @@ To get started, you can find several example Automation scripts with more inform
 
 After you have made sure that the [prerequisites](https://github.com/SkylineCommunications/ChatOps-Extensions/blob/main/ChatIntegrationExamples/README.md#usage) are in place, you can deploy [the Chat Integration examples](https://github.com/SkylineCommunications/ChatOps-Extensions/blob/main/ChatIntegrationExamples/README.md#getting-started) to your DataMiner System and immediately try out these examples.
 
+#### 6 February 2024 - New feature - Custom commands executed with the DataMiner bot now support adding buttons to the response using Adaptive Cards [ID_38701]
```
Comment (MariekeGO) [inline release-notes/Cloud/DCP_change_log.md:59]: This RN had already been included (see line 40-57). It seems odd to add this twice. Should the original entry be adjusted instead?

Comment (JordyGit) [reply release-notes/Cloud/DCP_change_log.md:59]: For us it was one change, but we didn't add it here yet explicitly for custom commands
Might be getting confusing if we mix them in one change log entry

Comment (MariekeGO) [reply release-notes/Cloud/DCP_change_log.md:59]: As there is so much overlap between the two entries, combining them seems quite doable. How about this?
```
#### 6 February 2024 - New feature - Chat Integration with Microsoft Teams now supports adding buttons in notifications and in responses using Adaptive Cards [ID_38701]

When using Adaptive Cards, Chat Integration now supports buttons in notifications to chats or channels and in custom command responses.

Available buttons:

- Prompt to run a custom command (optionally with predefined input values)
- Open URL
- Get element
- Get alarms of an element
- Get alarms of a view
- Prompt to change the active DMS of the conversation

In an Automation script, you can use [the DcpChatIntegrationHelper NuGet](https://www.nuget.org/packages/Skyline.DataMiner.DcpChatIntegrationHelper) to easily interact with Microsoft Teams.

To add a command to your DMS, create an Automation script in the folder "bot" in the DMS. For examples of such scripts, refer to [Custom Command Examples](https://github.com/SkylineCommunications/ChatOps-Extensions/tree/main/CustomCommandExamples) on GitHub.

You can also find several other example Automation scripts with more information on [GitHub](https://github.com/SkylineCommunications/ChatOps-Extensions/blob/main/ChatIntegrationExamples).

After you have made sure that the [prerequisites](https://github.com/SkylineCommunications/ChatOps-Extensions/blob/main/ChatIntegrationExamples/README.md#usage) are in place, you can deploy [the Chat Integration examples](https://github.com/SkylineCommunications/ChatOps-Extensions/blob/main/ChatIntegrationExamples/README.md#getting-started) to your DataMiner System and immediately try out these examples.

For more detailed information, refer to [Adding commands for the Teams bot to a DMS](xref:DataMiner_Teams_bot#adding-custom-commands-for-the-teams-bot-to-a-dms).
```

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**